tuplex/codegen/include/ASTAnnotation.h

```diff
@@ -400,6 +403,15 @@ class ASTAnnotation {
     ///! iterator-specific info
     std::shared_ptr<IteratorInfo> iteratorInfo;
 
+    ///! annotation for analyzing type-stability
+    size_t typeStableCount; // types are stable from the first to second last iteration (since type change in the last iteration is not a problem)
```

Comment:
maybe add above these a short documentation snippet/example what we refer to as type stability.

Reply:
Added examples.